libs/designsystem/checkbox/src/checkbox.component.scss

```diff
@@ -125,27 +122,51 @@ $spacing-to-label: map.get(utils.$checkbox-radio-spacing, 'to-label');
   &.has-label {
     display: block;
 
-    @each $size, $height in utils.$checkbox-radio-sizes {
+    @each $size, $properties in utils.$checkbox-sizes {
+      :host-context(.checkbox-#{$size}),
```

Comment:
This should not be necessary, as far as I remember it was an old cookbook thing that leaked into our implementation

```suggestion
```